### ISSUE TITLE
Mida.login error makes object undefined

### ISSUE THREAD

**Issue by dbvcode** [Reiryoku-Technologies/Mida#38]
I found out that when I call `Mida.login()` twice or multiple times on the cTrader account I get an error like:

```
{
  payloadType: 'PROTO_OA_ERROR_RES',
  ctidTraderAccountId: '***',
  errorCode: 'ALREADY_LOGGED_IN',
  description: 'Trading account is already authorized in this channel',
  maintenanceEndTimestamp: null,
  clientMsgId: '84c939f0-ee05-11ec-8fc5-a50cce82f4dc'
}
```

Code to exemplify:
```
let account = await  Mida.login("cTrader", {...}); //good
account = await  Mida.login("cTrader", {...}); //error
account = await  Mida.login("cTrader", {...}); //same error
```
At this point cTrader knows it's a connection but the `account` object is `undefined`.
I would expect that even with this error message the original/first object from the first login to be a functional connection still.
For some reason it's not. Even if I call it multiple times I get the same error.
You can even try like:
```
const account1 = await  Mida.login("cTrader", {...}); //good
const account2 = await  Mida.login("cTrader", {...}); //error
const account3 = await  Mida.login("cTrader", {...}); //same error
```
The initial `account` is undefined after the second login that logs the error.

**Comment by dbvcode** [comment]
I try to login into the same account

**Comment by Vasile-Peste** [comment]
Addressed in https://github.com/Reiryoku-Technologies/Mida-cTrader/pull/42

**Comment by Vasile-Peste** [comment]
Fixed with [Mida cTrader 5.4](https://github.com/Reiryoku-Technologies/Mida-cTrader/releases/tag/v5.4.0). Now logging into the same account returns the same cached instance.
